isis/tests/FunctionalTestsSocetLineScanKeywords.cpp

```diff
@@ -0,0 +1,152 @@
+#include "Fixtures.h"
+#include "Pvl.h"
+#include "PvlGroup.h"
+#include "TestUtilities.h"
+#include "Histogram.h"
+
+#include "socetlinescankeywords.h"
+
+#include "TestUtilities.h"
+#include "gtest/gtest.h"
+#include "gmock/gmock.h"
+
+using namespace Isis;
+using namespace testing; 
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/socetlinescankeywords.xml").expanded();
+
+TEST_F(MroCtxCube, FunctionalTestSocetLineScanKeywordsHirise) {
+  
+  QTemporaryDir prefix;
```

Comment:
Still unsure why we need another temporary directory here. The Inheritance chain for the MroCtxCube should also inherit the tempDir temporary directory, so we shouldn't need to define antoher temporary directory.

Reply:
I think there was a trend of tests having there own prefix but I do not know where that originated from so this was me following the formula. I can try without and see if it breaks anything.

Reply:
Hmmmm, weird. I don't really know why we would want/need separate input and output directories?

Reply:
Nothing failed locally, so I think it's fine 🤷‍♂️